ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -4,6 +4,9 @@
 
 fabric_name: DC1_FABRIC
 
+# If a device "is_deployed: false" at host_vars level, then shutdown facing peer ports.
+is_deployed_shutdown_peer_interfaces: true
```

Comment:
I would probably also rename the variable to make it clearer what it does e.g.
```
shutdown_interfaces_towards_undeployed_peers
```

Reply:
Agreed, originally I wanted the variable name to include "is_deployed" but there is no pretty way of doing this.